### ISSUE TITLE
Crash on import

### ISSUE THREAD

**Issue by FastusBellrune** [olivierkes/manuskript#1215]
The issue came about when I was importing a file.
I had a few .odt files to import with Pandoc and for one of those I accidentally forgot to specify the format, so it was left as Markdown.
When the window to choose the file opened, instead of only showing .md and .txt files, it showed all formats, and when I clicked on a .odt and tried to open it, the program crashed.

**Comment by TheJackiMonster** [comment]
Well it seems like you tried to import a Markdown file but selected an OpenDocument file. Beware the import dialog shows the currently selected format on the upper left before you select any file via the file dialog. I agree that the file dialog should filter files depending on that format. Also it would be better to not crash the application on wrong file selection.

But otherwise this seems to be user error. If you have Pandoc installed, you should be able to select ODT as format before selecting an .odt file for import. Hope this helps.

**Comment by FastusBellrune** [comment]
Yes, I knew that. I thought it was better to remove the option to choose files from different formats though, since mistakes happen. Thanks for the explanation anyway.
